aytests/tftp.list

```diff
@@ -2,4 +2,5 @@ tftp.sh                # tftp is running correctly
 host.sh                # host is in /etc/hosts (bsc#870998)
 unsupported_modules.sh # unsupported YaST modules were reported (bsc#925381)
 set_passwords.sh       # password are set (bsc#973639, bsc#974220, bsc#971804 and bsc#965852)
-post_script.sh         # download and execute post scripts (bnc#1014859)
\ No newline at end of file
+post_script.sh         # download and execute post scripts (bnc#1014859)
+ntp_conf.sh            # setting written to /etc/ntp.conf (bnc#1040350)
```

Comment:
I do not want to overload the sles12 test. Just want to distribute them to other tests....